pkg/sdn/api/v1beta3/types.go

```diff
@@ -35,3 +35,19 @@ type HostSubnetList struct {
 	kapi.ListMeta `json:"metadata,omitempty"`
 	Items         []HostSubnet `json:"items"`
 }
+
+// NetNamespace encapsulates the inputs needed to define a unique network namespace on the cluster
+type NetNamespace struct {
+	kapi.TypeMeta   `json:",inline"`
+	kapi.ObjectMeta `json:"metadata,omitempty"`
+
+	NetName string `json:"host" description:"Name of the network namespace."`
```

Comment:
Not any different right now.
At some point I was thinking about clients be able to choose the name of the network they belong to (and be allowed to change it later too). And the 'name' can stay as a uuid, or vice versa.